app/assets/styles/customization/espoo/customization.scss

```diff
@@ -29,25 +29,34 @@
   // Navbar
   // ------
 
-  .navbar-inverse {
+  .app-SideNavbar .app-Sidebar__sidebar {
     background-color: $blue;
 
-    .navbar-brand {
-      font-weight: $font-weight-bold;
-    }
+    .app-Navbar {
+      background-color: $blue;
 
-    .navbar-nav {
-      > .active > a,
-      > .active > a:hover,
-      > .active > a:focus {
-        background-color: darken($blue, 6.5%);
+      .main-nav {
+        h1, h2, h3, h4, h5, h6, p, a {
+          color: $white;
+        }
+        .nav.nav-stacked {
+          li {
+            border-bottom: 1px solid lighten($blue, 6);
+            &:first-of-type {
+              border-top: 1px solid lighten($blue, 6);
+            }
+          }
+        }
       }
+    }
+  }
 
-      .open > a,
-      .open > a:hover,
-      .open > a:focus {
-        background-color: darken($blue, 6.5%);
-      }
+  .app-SideNavbar__toggle {
+    background-color: $blue;
+    color: $white;
+
+    &:hover, &:focus {
+      color: $white;
     }
```

Comment:
This should also have styles for anchor colors, so "Close" button is white. Currently it has the same color than the background